docs/build.gradle

```diff
@@ -1440,6 +1440,95 @@ setups['calendar_outages_addevent'] = setups['calendar_outages_addjob'] + '''
            ]}
 '''
 
+// Used by the retrievers-examples
+  buildRestTests.setups['retrievers_examples'] = '''
```

Comment:
Normally these days we just do the test setup directly in the doc itself

see this [recent PR for example](https://github.com/elastic/elasticsearch/pull/115696/files#diff-2ed438e8b0249a1f0806ba278f579ae90da9e41a9b2135989dbbd3e55c37276dR107)